samples/Azure.Management.Storage/Generated/Extensions/ResourceGroupExtensionClient.cs

```diff
@@ -32,9 +32,9 @@ private string GetApiVersionOrNull(ResourceType resourceType)
             return apiVersion;
         }
 
-        /// <summary> Gets a collection of StorageAccounts in the StorageAccount. </summary>
-        /// <returns> An object representing collection of StorageAccounts and their operations over a StorageAccount. </returns>
-        public virtual StorageAccountCollection GetStorageAccounts()
+        /// <summary> Gets a collection of StorageAccountResources in the StorageAccountResource. </summary>
+        /// <returns> An object representing collection of StorageAccountResources and their operations over a StorageAccountResource. </returns>
+        public virtual StorageAccountCollection GetStorageAccountResources()
```

Comment:
We also need a decision on this: Do we need the Resource suffix after this function? I personally suppose `GetStorageAccounts` is more readable, especially this method is not returning a `StorageAccountResource` instance.